docker-storage-setup.sh

```diff
@@ -1112,6 +1112,15 @@ fi
 if [ -e /etc/sysconfig/docker-storage-setup ]; then
   source /etc/sysconfig/docker-storage-setup
 fi
+if [ -z "$ROOT_VOLUME" ]; then
+    ROOT_VOLUME="$DOCKER_ROOT_VOLUME"
+fi
```

Comment:
Good catch, I have reversed the logic.  If a user sets a DOCKER_ variable it will override.